example/src/main/java/com/stripe/example/StripeFactory.kt

```diff
@@ -10,11 +10,13 @@ internal class StripeFactory(
     private val enableLogging: Boolean = true
 ) {
     fun create(): Stripe {
+        val paymentConfiguration = PaymentConfiguration.getInstance(context)
         return Stripe(
```

Comment:
We really should offer an overloaded `Stripe` constructor that takes a `PaymentConfiguration` instance.